src/index.browser.js

```diff
@@ -48,19 +49,39 @@ window.onload = () => {
 
 }
 
-ReactDOM.render(
+const render = (Root: React$Element<*>) => {
 
+  ReactDOM.render(
+    <AppContainer>
+      { Root }
+    </AppContainer>,
+    document.getElementById('react-mount')
+  )
+
+}
+
+const App = (
   <Provider store={store}>
     <MuiThemeProvider muiTheme={getMuiTheme()}>
       <Router>
         <Routes />
       </Router>
     </MuiThemeProvider>
-  </Provider>,
-
-  document.getElementById('react-mount')
+  </Provider>
 )
 
+render(App)
+
+if (module.hot) {
+
+  module.hot.accept('src/routes', () => {
+
+    render(App)
+
+  })
+
+}
```

Comment:
Did you try out the new relaxed arrow function lint rules??  ...we can now do this: 😉 

```js
if (module.hot) module.hot.accept('src/routes', () => render(App))
```